--- sundials-2.6.2/src/arkode/CMakeLists.txt.orig	2015-07-31 02:07:37.000000000 +0200
+++ sundials-2.6.2/src/arkode/CMakeLists.txt	2023-01-19 17:32:41.649626680 +0100
@@ -127,7 +127,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_arkode CLEAN_DIRECT_OUTPUT 1)
 
   # Install the ARKODE library
-  INSTALL(TARGETS sundials_arkode_static DESTINATION lib)
+  INSTALL(TARGETS sundials_arkode_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -147,7 +147,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${arkodelib_VERSION} SOVERSION ${arkodelib_SOVERSION})
 
   # Install the ARKODE library
-  INSTALL(TARGETS sundials_arkode_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_arkode_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-2.6.2/src/cvode/CMakeLists.txt.orig	2015-07-31 02:08:00.000000000 +0200
+++ sundials-2.6.2/src/cvode/CMakeLists.txt	2023-01-19 17:34:07.565827899 +0100
@@ -118,7 +118,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_cvode CLEAN_DIRECT_OUTPUT 1)
 
   # Install the CVODE library
-  INSTALL(TARGETS sundials_cvode_static DESTINATION lib)
+  INSTALL(TARGETS sundials_cvode_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -138,7 +138,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${cvodelib_VERSION} SOVERSION ${cvodelib_SOVERSION})
 
   # Install the CVODE library
-  INSTALL(TARGETS sundials_cvode_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_cvode_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-3.1.1/src/nvec_ser/CMakeLists.txt.orig	2018-05-08 03:19:20.000000000 +0200
+++ sundials-3.1.1/src/nvec_ser/CMakeLists.txt	2023-01-21 21:40:58.202787181 +0100
@@ -43,7 +43,7 @@ IF(BUILD_STATIC_LIBS)
   ADD_LIBRARY(sundials_nvecserial_static STATIC ${nvecserial_SOURCES} ${shared_SOURCES})
   SET_TARGET_PROPERTIES(sundials_nvecserial_static
     PROPERTIES OUTPUT_NAME sundials_nvecserial CLEAN_DIRECT_OUTPUT 1)
-  INSTALL(TARGETS sundials_nvecserial_static DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecserial_static DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_STATIC_LIBS)
 
 # Rules for building and installing the shared library:
@@ -62,7 +62,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES OUTPUT_NAME sundials_nvecserial CLEAN_DIRECT_OUTPUT 1)
   SET_TARGET_PROPERTIES(sundials_nvecserial_shared
     PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-  INSTALL(TARGETS sundials_nvecserial_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecserial_shared DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_SHARED_LIBS)
 
 # Install the NVECSERIAL header files
@@ -76,7 +76,7 @@ IF(FCMIX_ENABLE AND F77_FOUND)
     ADD_LIBRARY(sundials_fnvecserial_static STATIC ${fnvecserial_SOURCES})
     SET_TARGET_PROPERTIES(sundials_fnvecserial_static
       PROPERTIES OUTPUT_NAME sundials_fnvecserial CLEAN_DIRECT_OUTPUT 1)
-    INSTALL(TARGETS sundials_fnvecserial_static DESTINATION lib)
+    INSTALL(TARGETS sundials_fnvecserial_static DESTINATION lib${LIB_SUFFIX})
   ENDIF(BUILD_STATIC_LIBS)
 
   IF(BUILD_SHARED_LIBS)
@@ -89,7 +89,7 @@ IF(FCMIX_ENABLE AND F77_FOUND)
       PROPERTIES OUTPUT_NAME sundials_fnvecserial CLEAN_DIRECT_OUTPUT 1)
     SET_TARGET_PROPERTIES(sundials_fnvecserial_shared 
       PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-    INSTALL(TARGETS sundials_fnvecserial_shared DESTINATION lib)
+    INSTALL(TARGETS sundials_fnvecserial_shared DESTINATION lib${LIB_SUFFIX})
   ENDIF(BUILD_SHARED_LIBS)
 
 ENDIF(FCMIX_ENABLE AND F77_FOUND)
--- sundials-3.1.1/src/nvec_openmp/CMakeLists.txt.orig	2018-05-08 03:19:20.000000000 +0200
+++ sundials-3.1.1/src/nvec_openmp/CMakeLists.txt	2023-01-21 21:40:26.789624027 +0100
@@ -50,7 +50,7 @@ IF(BUILD_STATIC_LIBS)
   ADD_LIBRARY(sundials_nvecopenmp_static STATIC ${nvecopenmp_SOURCES} ${shared_SOURCES})
   SET_TARGET_PROPERTIES(sundials_nvecopenmp_static
     PROPERTIES OUTPUT_NAME sundials_nvecopenmp CLEAN_DIRECT_OUTPUT 1)
-  INSTALL(TARGETS sundials_nvecopenmp_static DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecopenmp_static DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_STATIC_LIBS)
 
 # Rules for building and installing the shared library:
@@ -69,7 +69,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES OUTPUT_NAME sundials_nvecopenmp CLEAN_DIRECT_OUTPUT 1)
   SET_TARGET_PROPERTIES(sundials_nvecopenmp_shared
     PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-  INSTALL(TARGETS sundials_nvecopenmp_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecopenmp_shared DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_SHARED_LIBS)
 
 # Install the NVECOPENMP header files
@@ -96,7 +96,7 @@ IF(FCMIX_ENABLE AND F77_FOUND)
       PROPERTIES OUTPUT_NAME sundials_fnvecopenmp CLEAN_DIRECT_OUTPUT 1)
     SET_TARGET_PROPERTIES(sundials_fnvecopenmp_shared 
       PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-    INSTALL(TARGETS sundials_fnvecopenmp_shared DESTINATION lib)
+    INSTALL(TARGETS sundials_fnvecopenmp_shared DESTINATION lib${LIB_SUFFIX})
   ENDIF(BUILD_SHARED_LIBS)
 
 ENDIF(FCMIX_ENABLE AND F77_FOUND)
--- sundials-3.1.1/src/nvec_pthreads/CMakeLists.txt.orig	2018-05-08 03:19:20.000000000 +0200
+++ sundials-3.1.1/src/nvec_pthreads/CMakeLists.txt	2023-01-21 21:41:47.525853309 +0100
@@ -43,7 +43,7 @@ IF(BUILD_STATIC_LIBS)
   ADD_LIBRARY(sundials_nvecpthreads_static STATIC ${nvecpthreads_SOURCES} ${shared_SOURCES})
   SET_TARGET_PROPERTIES(sundials_nvecpthreads_static
     PROPERTIES OUTPUT_NAME sundials_nvecpthreads CLEAN_DIRECT_OUTPUT 1)
-  INSTALL(TARGETS sundials_nvecpthreads_static DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecpthreads_static DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_STATIC_LIBS)
 
 # Rules for building and installing the shared library:
@@ -64,7 +64,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES OUTPUT_NAME sundials_nvecpthreads CLEAN_DIRECT_OUTPUT 1)
   SET_TARGET_PROPERTIES(sundials_nvecpthreads_shared
     PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-  INSTALL(TARGETS sundials_nvecpthreads_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_nvecpthreads_shared DESTINATION lib${LIB_SUFFIX})
 ENDIF(BUILD_SHARED_LIBS)
 
 # Install the NVECPTHREADS header files
@@ -78,7 +78,7 @@ IF(FCMIX_ENABLE AND F77_FOUND)
     ADD_LIBRARY(sundials_fnvecpthreads_static STATIC ${fnvecpthreads_SOURCES})
     SET_TARGET_PROPERTIES(sundials_fnvecpthreads_static
       PROPERTIES OUTPUT_NAME sundials_fnvecpthreads CLEAN_DIRECT_OUTPUT 1)
-    INSTALL(TARGETS sundials_fnvecpthreads_static DESTINATION lib)
+    INSTALL(TARGETS sundials_fnvecpthreads_static DESTINATION lib${LIB_SUFFIX})
   ENDIF(BUILD_STATIC_LIBS)
 
   IF(BUILD_SHARED_LIBS)
@@ -91,7 +91,7 @@ IF(FCMIX_ENABLE AND F77_FOUND)
       PROPERTIES OUTPUT_NAME sundials_fnvecpthreads CLEAN_DIRECT_OUTPUT 1)
     SET_TARGET_PROPERTIES(sundials_fnvecpthreads_shared 
       PROPERTIES VERSION ${nveclib_VERSION} SOVERSION ${nveclib_SOVERSION})
-    INSTALL(TARGETS sundials_fnvecpthreads_shared DESTINATION lib)
+    INSTALL(TARGETS sundials_fnvecpthreads_shared DESTINATION lib${LIB_SUFFIX})
   ENDIF(BUILD_SHARED_LIBS)
 ENDIF(FCMIX_ENABLE AND F77_FOUND)
 
--- sundials-2.6.2/src/cvodes/CMakeLists.txt.orig	2015-07-31 02:08:15.000000000 +0200
+++ sundials-2.6.2/src/cvodes/CMakeLists.txt	2023-01-20 06:14:20.281121648 +0100
@@ -121,7 +121,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_cvodes CLEAN_DIRECT_OUTPUT 1)
 
   # Install the CVODES library
-  INSTALL(TARGETS sundials_cvodes_static DESTINATION lib)
+  INSTALL(TARGETS sundials_cvodes_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -141,7 +141,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${cvodeslib_VERSION} SOVERSION ${cvodeslib_SOVERSION})
 
   # Install the CVODES library
-  INSTALL(TARGETS sundials_cvodes_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_cvodes_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-2.6.2/src/ida/CMakeLists.txt.orig	2015-07-31 02:08:39.000000000 +0200
+++ sundials-2.6.2/src/ida/CMakeLists.txt	2023-01-20 06:15:08.862858469 +0100
@@ -114,7 +114,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_ida CLEAN_DIRECT_OUTPUT 1)
 
   # Install the IDA library
-  INSTALL(TARGETS sundials_ida_static DESTINATION lib)
+  INSTALL(TARGETS sundials_ida_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -134,7 +134,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${idalib_VERSION} SOVERSION ${idalib_SOVERSION})
 
   # Install the IDA library
-  INSTALL(TARGETS sundials_ida_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_ida_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-2.6.2/src/idas/CMakeLists.txt.orig	2015-07-31 02:08:55.000000000 +0200
+++ sundials-2.6.2/src/idas/CMakeLists.txt	2023-01-20 06:15:52.131274057 +0100
@@ -126,7 +126,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_idas CLEAN_DIRECT_OUTPUT 1)
 
   # Install the IDA library
-  INSTALL(TARGETS sundials_idas_static DESTINATION lib)
+  INSTALL(TARGETS sundials_idas_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -146,7 +146,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${idaslib_VERSION} SOVERSION ${idaslib_SOVERSION})
 
   # Install the IDAS library
-  INSTALL(TARGETS sundials_idas_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_idas_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-2.7.0/src/kinsol/CMakeLists.txt.orig	2016-09-27 19:27:29.000000000 +0200
+++ sundials-2.7.0/src/kinsol/CMakeLists.txt	2023-01-21 07:57:43.080383525 +0100
@@ -134,7 +134,7 @@ IF(BUILD_STATIC_LIBS)
     PROPERTIES OUTPUT_NAME sundials_kinsol CLEAN_DIRECT_OUTPUT 1)
 
   # Install the KINSOL library
-  INSTALL(TARGETS sundials_kinsol_static DESTINATION lib)
+  INSTALL(TARGETS sundials_kinsol_static DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_STATIC_LIBS)
 
@@ -154,7 +154,7 @@ IF(BUILD_SHARED_LIBS)
     PROPERTIES VERSION ${kinsollib_VERSION} SOVERSION ${kinsollib_SOVERSION})
 
   # Install the KINSOL library
-  INSTALL(TARGETS sundials_kinsol_shared DESTINATION lib)
+  INSTALL(TARGETS sundials_kinsol_shared DESTINATION lib${LIB_SUFFIX})
 
 ENDIF(BUILD_SHARED_LIBS)
 
--- sundials-2.7.0/src/arkode/fcmix/CMakeLists.txt.orig	2016-09-27 19:27:29.000000000 +0200
+++ sundials-2.7.0/src/arkode/fcmix/CMakeLists.txt	2023-01-21 21:23:53.951669359 +0100
@@ -70,7 +70,7 @@ SET_TARGET_PROPERTIES(sundials_farkode_s
   PROPERTIES OUTPUT_NAME sundials_farkode CLEAN_DIRECT_OUTPUT 1)
 
 # Install the FARKODE library
-INSTALL(TARGETS sundials_farkode_static DESTINATION lib)
+INSTALL(TARGETS sundials_farkode_static DESTINATION lib${LIB_SUFFIX})
 
 #
 MESSAGE(STATUS "Added ARKODE FCMIX module")
--- sundials-2.7.0/src/cvode/fcmix/CMakeLists.txt.orig	2016-09-27 19:27:29.000000000 +0200
+++ sundials-2.7.0/src/cvode/fcmix/CMakeLists.txt	2023-01-21 21:24:19.988194974 +0100
@@ -57,7 +57,7 @@ SET_TARGET_PROPERTIES(sundials_fcvode_st
   PROPERTIES OUTPUT_NAME sundials_fcvode CLEAN_DIRECT_OUTPUT 1)
 
 # Install the FCVODE library
-INSTALL(TARGETS sundials_fcvode_static DESTINATION lib)
+INSTALL(TARGETS sundials_fcvode_static DESTINATION lib${LIB_SUFFIX})
 
 #
 MESSAGE(STATUS "Added CVODE FCMIX module")
--- sundials-2.7.0/src/ida/fcmix/CMakeLists.txt.orig	2016-09-27 19:27:29.000000000 +0200
+++ sundials-2.7.0/src/ida/fcmix/CMakeLists.txt	2023-01-21 21:24:39.378089930 +0100
@@ -56,6 +56,6 @@ SET_TARGET_PROPERTIES(sundials_fida_stat
   PROPERTIES OUTPUT_NAME sundials_fida CLEAN_DIRECT_OUTPUT 1)
 
 # Install the FIDA library
-INSTALL(TARGETS sundials_fida_static DESTINATION lib)
+INSTALL(TARGETS sundials_fida_static DESTINATION lib${LIB_SUFFIX})
 #
 MESSAGE(STATUS "Added IDA FCMIX module")
--- sundials-2.7.0/src/kinsol/fcmix/CMakeLists.txt.orig	2016-09-27 19:27:29.000000000 +0200
+++ sundials-2.7.0/src/kinsol/fcmix/CMakeLists.txt	2023-01-21 21:24:56.191332178 +0100
@@ -54,7 +54,7 @@ SET_TARGET_PROPERTIES(sundials_fkinsol_s
   PROPERTIES OUTPUT_NAME sundials_fkinsol CLEAN_DIRECT_OUTPUT 1)
 
 # Install the FKINSOL library
-INSTALL(TARGETS sundials_fkinsol_static DESTINATION lib)
+INSTALL(TARGETS sundials_fkinsol_static DESTINATION lib${LIB_SUFFIX})
 
 #
 MESSAGE(STATUS "Added KINSOL FCMIX module")
