--- sundials-2.3.0/src/cvode/cvode.c.orig	2006-11-07 00:18:34.000000000 +0100
+++ sundials-2.3.0/src/cvode/cvode.c	2023-01-17 17:08:18.237375520 +0100
@@ -4034,7 +4034,7 @@ void CVProcessError(CVodeMem cv_mem,
 
 #ifndef NO_FPRINTF_OUTPUT
     fprintf(stderr, "\n[%s ERROR]  %s\n  ", module, fname);
-    fprintf(stderr, msgfmt);
+    vfprintf(stderr, msgfmt, ap);
     fprintf(stderr, "\n\n");
 #endif
 
--- sundials-2.3.0/src/cvodes/cvodes.c.orig	2006-11-07 00:18:34.000000000 +0100
+++ sundials-2.3.0/src/cvodes/cvodes.c	2023-01-17 17:08:55.410518470 +0100
@@ -7185,7 +7185,7 @@ void CVProcessError(CVodeMem cv_mem,
 
 #ifndef NO_FPRINTF_OUTPUT
     fprintf(stderr, "\n[%s ERROR]  %s\n  ", module, fname);
-    fprintf(stderr, msgfmt);
+    vfprintf(stderr, msgfmt, ap);
     fprintf(stderr, "\n\n");
 #endif
 
--- sundials-2.3.0/src/ida/ida.c.orig	2006-11-07 00:18:34.000000000 +0100
+++ sundials-2.3.0/src/ida/ida.c	2023-01-17 17:09:29.833645317 +0100
@@ -3152,7 +3152,7 @@ void IDAProcessError(IDAMem IDA_mem,
 
 #ifndef NO_FPRINTF_OUTPUT
     fprintf(stderr, "\n[%s ERROR]  %s\n  ", module, fname);
-    fprintf(stderr, msgfmt);
+    vfprintf(stderr, msgfmt, ap);
     fprintf(stderr, "\n\n");
 #endif
 
--- sundials-2.3.0/src/kinsol/kinsol.c.orig	2006-11-07 00:18:34.000000000 +0100
+++ sundials-2.3.0/src/kinsol/kinsol.c	2023-01-17 17:09:50.793551767 +0100
@@ -1858,7 +1858,7 @@ void KINProcessError(KINMem kin_mem,
 
 #ifndef NO_FPRINTF_OUTPUT
     fprintf(stderr, "\n[%s ERROR]  %s\n  ", module, fname);
-    fprintf(stderr, msgfmt);
+    vfprintf(stderr, msgfmt, ap);
     fprintf(stderr, "\n\n");
 #endif
 
